tmt/steps/provision/local.py

```diff
@@ -27,6 +27,13 @@ class GuestLocal(tmt.Guest):
     localhost = True
     parent: Optional[tmt.steps.Step]
 
+    @property
+    def scripts_path(self) -> Path:
+        """
+        Absolute path to tmt scripts directory
+        """
+        return tmt.steps.scripts.effective_scripts_dest_dir(default=Path("/var/lib/tmt/scripts"))
```

Comment:
Right, and the path itself should be a module-level constant in this plugin, just like https://github.com/teemtee/tmt/blob/main/tmt/steps/scripts.py#L24 and https://github.com/teemtee/tmt/blob/main/tmt/steps/scripts.py#L27 are defined in `tmt.steps.scripts` for the general usage.